dockers/docker-teamd/supervisord.conf

```diff
@@ -44,6 +44,7 @@ command=/usr/bin/teammgrd
 priority=3
 autostart=false
 autorestart=false
+stopwaitsecs=60
```

Comment:
Do you see teamsyncd taking longer time on this testbed ? if that is the case we can add this setting "stopwaitsecs" for teamsyncd as well.

Reply:
@judyjoseph i did some investigation regarding that and it seems that we don't need to increase the timeout for `teamsyncd`:

```bash
root@sonic:/# supervisorctl start teamsyncd

root@sonic:/home/admin# cat /var/log/syslog | grep teamsyncd
May 27 16:51:24.050896 sonic INFO teamd#supervisord 2021-05-27 16:51:24,050 INFO spawned: 'teamsyncd' with pid 370
May 27 16:51:24.060687 sonic NOTICE teamd#teamsyncd: :- loadRedisScript: lua script loaded, sha: 88270a7c5c90583e56425aca8af8a4b8c39fe757
May 27 16:51:24.065430 sonic INFO teamd#supervisord: teamsyncd Listens to teamd events...
May 27 16:51:24.172000 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.200503 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list:
May 27 16:51:24.200995 sonic INFO teamd#supervisord: teamsyncd get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.376369 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.411137 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.438195 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.470041 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.957463 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:25.051005 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:25.200879 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:29.396578 sonic INFO teamd#supervisord 2021-05-27 16:51:29,396 INFO success: teamsyncd entered RUNNING state, process has stayed up for > than 5 seconds (startsecs)
```

**diff:** May 27 16:51:29.396578 - May 27 16:51:24.050896 => 5 sec

```bash
root@sonic:/# supervisorctl stop teamsyncd

May 27 16:52:48.727690 sonic INFO teamd#supervisord 2021-05-27 16:52:48,727 INFO waiting for teamsyncd to stop
May 27 16:52:49.728149 sonic NOTICE teamd#teamsyncd: :- cleanTeamSync: Cleaning up LAG teamd resources ...
May 27 16:52:49.765961 sonic NOTICE teamd#teamsyncd: :- main: Received SIGTERM Exiting
May 27 16:52:49.781918 sonic INFO teamd#supervisord 2021-05-27 16:52:49,781 INFO stopped: teamsyncd (exit status 0)
```

**diff:** May 27 16:52:49.781918 - May 27 16:52:48.727690 => 1 sec